doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -200,19 +200,20 @@ Bluetooth samples
 * :ref:`peripheral_hids_mouse` and :ref:`central_uart` samples now come with the :ref:`ble_rpc_host` child image configuration overlay.
   The overlay shows how to configure an application running a serialized Bluetooth Low Energy Host.
   These samples run out the box with the :ref:`ble_rpc` library.
+* Added :ref:`central_and_peripheral_hrs` sample.
 
 Bluetooth mesh samples
 ----------------------
 
 * Added:
 
-  * :ref:`bluetooth_ble_peripheral_lbs_coex` sample, demonstrating how to combine Bluetooth mesh and Bluetooth Low Energy features in a single application.
+  * :ref:`bluetooth_ble_peripheral_lbs_coex`, demonstrating how to combine Bluetooth mesh and Bluetooth Low Energy features in a single application.
   * Support for :ref:`zephyr:nrf21540dk_nrf52840`.
-  * :ref:`central_and_peripheral_hrs` sample.
+  * :ref:`bluetooth_mesh_light_lc` and :ref:`bluetooth_mesh_sensor_server` with support for :ref:`zephyr:thingy53_nrf5340`.
```

Comment:
These samples weren't added in this release, and should probably be grouped with the light and light switch under "updated"